templates/default/tags/html/option.erb

```diff
@@ -1,4 +1,4 @@
-<% if object.has_tag?(:option) %>
+<% if object.has_tag?(:option) && object.type == :method %>
```

Comment:
This might be better as a `respond_to?(:parameters)` check, since other (custom) types might support option tags.

Reply:
OK. I'll fix later.